packages/uuuu/uuuu.0.2.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name:         "uuuu"
```

Comment:
```suggestion
```
It will be overridden by opam anyways.

We usually remove these on merge but there is a small change so it could be done in the same commit